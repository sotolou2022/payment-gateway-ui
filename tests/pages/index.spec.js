import { mount, enableAutoUnmount } from "@vue/test-utils";
import { beforeEach, afterEach, expect } from "@jest/globals";
import Component from "~~/index";
import authPlugin from "~/plugins/auth";
import { createTestingPinia } from "@pinia/testing";

let wrapper;

beforeEach(() => {
  wrapper = mount(Component, {
    global: { plugins: [createTestingPinia(), authPlugin] },
    shallow: true,
  });
});

enableAutoUnmount(afterEach);

describe("Test suite for index page", () => {
  it("is a vue component", () => expect(wrapper.vm).toBeTruthy());

  it("match snapshot", () => expect(wrapper.html()).toMatchSnapshot());
});
