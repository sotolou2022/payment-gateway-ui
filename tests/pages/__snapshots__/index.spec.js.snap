// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for index page match snapshot 1`] = `
"<transition-stub>
  <div class=\\"col-12 col-md-8\\">
    <div class=\\"card shadow rounded-lg py-5\\">
      <div class=\\"card-body\\">
        <div class=\\"row align-items-center\\">
          <anonymous-stub class=\\"col-xl-6\\"></anonymous-stub>
          <anonymous-stub class=\\"col-xl-6\\"></anonymous-stub>
        </div>
      </div>
    </div>
  </div>
</transition-stub>"
`;
